dspace-api/src/main/java/org/dspace/authorize/service/ResourcePolicyService.java

```diff
@@ -66,26 +67,60 @@
     public void removePolicies(Context c, DSpaceObject o, String type) throws SQLException, AuthorizeException;
 
     public void removeDsoGroupPolicies(Context context, DSpaceObject dso, Group group)
-        throws SQLException, AuthorizeException;
+            throws SQLException, AuthorizeException;
 
     public void removeDsoEPersonPolicies(Context context, DSpaceObject dso, EPerson ePerson)
-        throws SQLException, AuthorizeException;
+            throws SQLException, AuthorizeException;
 
     public void removeGroupPolicies(Context c, Group group) throws SQLException;
 
     public void removeDsoAndTypeNotEqualsToPolicies(Context c, DSpaceObject o, String type)
-        throws SQLException, AuthorizeException;
+            throws SQLException, AuthorizeException;
 
     /**
      * Return a list of policies for an object that match the action except the record labeled with the rpType
      *
-     * @param c        context
-     * @param o        DSpaceObject policies relate to
-     * @param actionID action (defined in class Constants)
-     * @param rpType   the resource policy type
+     * @param c
+     *            context
+     * @param o
+     *            DSpaceObject policies relate to
+     * @param actionID
+     *            action (defined in class Constants)
+     * @param rpType
+     *            the resource policy type
      * @return list of resource policies
-     * @throws SQLException if there's a database problem
+     * @throws SQLException
+     *             if there's a database problem
      */
     public List<ResourcePolicy> findExceptRpType(Context c, DSpaceObject o, int actionID, String rpType)
             throws SQLException;
+
+    public List<ResourcePolicy> findByEPerson(Context c, EPerson ePerson, int offset, int limit) throws SQLException;
+
+    public int searchCountEPerson(Context context, EPerson eperson) throws SQLException;
```

Comment:
Similar to my comment about renaming `searchBy` to `findBy`, these `searchCount` methods could probably be renamed as `count` or `countBy` methods (they don't need the word "search").  See for example, other `count` methods in other Services (e.g. `ItemService`)